app/services/moves/finder.rb

```diff
@@ -59,6 +58,12 @@ def apply_filters(scope)
       scope
     end
 
+    def apply_includes(scope)
+      scope = scope.includes(db_includes)
+      # scope = scope.joins(db_includes) # NB: do not join - you may get duplicates
```

Comment:
Remove commented line?

Reply:
You can do this all on one line too, so maybe the method is superfluous and could be replaced on line 22 with

```ruby
scope = scope.includes(db_includes)
```